site/guides/namespace-provisioning-cli.md

```diff
@@ -1,4 +1,4 @@
-# Namespace provisioning example via kpt CLI
+# Namespace provisioning using kpt CLI
```

Comment:
Do we want to align the naming between CLI and UI (UI is marked as "example" while CLI is not)